src/Resources/views/Shop/Product/index.html.twig

```diff
@@ -59,3 +56,54 @@
     </div>
     {{ form_end(form, {'render_rest': false}) }}
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script type="application/javascript">
```

Comment:
Fan we make this file a public JS asset as a jQuery plugin? IMHO having that long JS in a Twit template makes the code a little bit hard to manage, especially if in the future someone is going to customize this template. WDYT?

Reply:
You're right. I will change this.

Reply:
Yup, that would way better soultion. Moved to separate JS file and made it as a jQuery plugin 🚀